ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/DisplayClusters/README.metadata.json

```diff
@@ -25,9 +25,9 @@
         "IdentifyLayerResult"
     ],
     "snippets": [
-        "DisplayPointsUsingClustering.cpp",
-        "DisplayPointsUsingClustering.h",
-        "DisplayPointsUsingClustering.qml"
+        "DisplayClusters.cpp",
```

Comment:
since this sample has already been published to the website, we need to fill out the "redirect_from" key to be `["qt/cpp/sample-code/display-points-using-clustering-feature-reduction"]`

This means that any doc or bookmarks pointing to the old URL will redirect to the new one